lib/trie/trie.go

```diff
@@ -515,6 +538,10 @@ func (t *Trie) insertInBranch(parentBranch *Node, key, value []byte) (
 // The keys are in hexadecimal little Endian encoding and the values
 // are hexadecimal encoded.
 func (t *Trie) LoadFromMap(data map[string]string) (err error) {
+	defer func() {
+		t.handleTrackedDeltas(err)
```

Comment:
I don't see how `err` is every set in this.  Won't it always be `nil`?

Reply:
No it will capture the returned named error, so if any `return xxx` happens it will get caught in that deferred function